examples/transform/src/main/java/org/apache/apex/examples/transform/DynamicTransformApplication.java

```diff
@@ -44,6 +48,22 @@
   private static String MAX_THROUGHPUT = "dt.maxThroughput";
   private static String MIN_THROUGHPUT = "dt.minThroughput";
 
+  @SuppressWarnings("serial")
+  public static final Function.MapFunction<Object, Void> CONSOLE_OUTPUT_FN = new Function.MapFunction<Object, Void>()
```

Comment:
Since it repeats in multiple places, make it resusable.

Reply:
Refactored to extend from the Simple version of this class.